src/kernel/mempool_persist.cpp

```diff
@@ -194,7 +196,7 @@ bool DumpMempool(const CTxMemPool& pool, const fs::path& dump_path, FopenFn mock
 
         file << mapDeltas;
 
-        LogPrintf("Writing %d unbroadcast transactions to disk.\n", unbroadcast_txids.size());
+        LogInfo("Writing %d unbroadcast transactions to file.\n", unbroadcast_txids.size());
```

Comment:
nit: this seems to be in the wrong commit

Reply:
Updated it is now in 4d5b55735bcf82847d748d24da5dbdbd1de8ef41